autoscale-policy.json

```diff
@@ -0,0 +1,22 @@
+{
+  "instance_min_count": 2,
+  "instance_max_count": 10,
+  "scaling_rules": [
+    {
+      "metric_type": "responsetime",
+      "breach_duration_secs": 120,
+      "threshold": 1000,
+      "operator": "<",
+      "cool_down_secs": 60,
+      "adjustment": "-1"
+    },
+    {
+      "metric_type": "responsetime",
+      "breach_duration_secs": 120,
+      "threshold": 1000,
```

Comment:
I'm wondering if these two `threshold` values should be different.
e.g. the first one could be 500 and the second one could be 1000.

I'm worried that, if the response time is hovering around 1 second, we'll constantly be adding and removing instances.

If we set the "scale down" threshold to be 500, then we're saying "only scale down once everything's running very smoothly again".

What do you think?

Reply:
I don't think 500ms is forgiving enough, sometimes the thing just runs slowly for a couple of seconds for no apparent reason...
If we get the policy in as-is for now so I can finish setup, then we can experiment from there